I worked on add 16Bit IN/OUT

parshwa282011 opened the pull request:
I added 16 Bit In/Out as it makes more sense for the ROM256x16 to be a 16Bit out instead of 2 8Bit or it could be a choice for someone for which one to use. I also added a RGBLED but it is very large as there are 8Bit Inputs so there is 0 to 255 range for each RED, GREEN, and BLUE.

parshwa282011 committed:
trying to make a 16 bit inputs and outputs

parshwa282011 committed:
Merge branch 'SebLague:main' into main

parshwa282011 committed:
Merge branch 'SebLague:main' into main

Commit:
RGBLED working + added 16 Bit Bus, and 16 Bit IN?OUT the 16 Bit merges and Splits arent working yet

Commit:
ALL 16-BIT merge and Splits work now

Commit:
fixed a few bugs

pk3zip commented:
damn i didnt know you were an editor i was reviewing changes

parshwa282011 committed:
Merge branch 'main' into merge

parshwa282011 committed:
Merging for new updates

Merge

parshwa282011 commented:
what i didn't close this i was trying to merge new updates and accedentaly pushed that pr here so i closed that not this one

sk337 commented on Assets/Scripts/Graphics/UI/Menus/RomEditMenu.cs:
```diff
@@ -210,42 +211,61 @@ static bool ValidateInputString(string text)
+			}catch (Exception e){
+				Debug.Log(e);
+				if (stringFormat is DataDisplayMode.Binary){
+					uintVal =(uint) ((ulong) (Convert.ToInt64(displayString, 2))& 0b11111111111111111111111111111111);
```
could you either replace this with 0xFFFFFFFF or a constant like UINT32_MAX

parshwa282011 replied:
i changed it to 0xFFFFFFFF

parshwa282011 committed:
changed 0b11111111111111111111111111111111 to 0xFFFFFFFF as suggested

parshwa282011 committed:
Removing vscode files

parshwa282011 committed:
Create main.yml

parshwa282011 committed:
My mistake i ment for this to be on test branch

Commit:
Works with Ascii 8bit
The only issue is special charachters are just blank plus 128-255 is also blank

parshwa282011 committed:
Merge pull request #3 from parshwa282011/tmp

Works with Ascii 8bit

H4SO6 commented:
Definitly needs to be added
